crates/turbo-tasks-fs/src/rope.rs

```diff
@@ -686,6 +702,22 @@ impl From<RopeElem> for StackElem {
     }
 }
 
+trait PtrEq {
```

Comment:
I don't think we need a new trait for this?
Could add a small helper function for `Bytes`